app.rb

```diff
@@ -0,0 +1,80 @@
+require 'date'
+require_relative './game'
+require_relative './musicalbum'
+require_relative './create'
+require_relative './list'
+
+class App
+  def run
+    check_files
+
+    @create_album = CreateMusicAlbum.new
+    @create_game = CreateGames.new
+    @list_albums = ListMusicAlbums.new
+    @list_genres = ListGenres.new
+    @list_games = ListGames.new
+    @list_authors = ListAuthors.new
+
+    puts 'Welcome to Catalog of Things!'
+    loop do
+      menu
+      option = gets.chomp.to_i
+      get_num(option)
+      next unless option == 10
+
+      puts '----'
+      print "Thank you for using the Catalog of Things!\n"
+      puts '----'
+      exit!
+    end
+  end
+
+  # check if file exist and creates file if not
+
+  def check_files
+    File.open('./files/musicalbums.json', 'a') unless File.exist?('./files/musicalbums.json')
+    File.open('./files/games.json', 'a') unless File.exist?('./files/games.json')
+    File.open('./files/books.json', 'a') unless File.exist?('./files/books.json')
+  end
+
+  def menu
+    puts
+    puts 'Please choose an option by entering a number'
+    puts '1 - List all books'
+    puts '2 - List all music albums'
+    puts '3 - List all games'
```

Comment:
- [ ] If remove all the data and then select option 3 it breaks and throws an error. It would be great to keep it like the other options. If we have not data raised a message telling that there is no data or something similar.

![Screen Shot 2022-03-25 at 9 02 56](https://user-images.githubusercontent.com/67211919/160146615-e2bb1128-d224-427a-9e06-6b0a3a2c2f11.png)